spec/support/ansible_shared/automation_manager/credential.rb

```diff
@@ -1,8 +1,8 @@
 require 'ansible_tower_client'
 
-shared_examples_for "ansible credential" do
+shared_examples_for "ansible credential" do |ansible_provider|
   let(:finished_task) { FactoryGirl.create(:miq_task, :state => "Finished") }
-  let(:manager)       { FactoryGirl.create(:provider_ansible_tower, :with_authentication).managers.first }
+  let(:manager)       { FactoryGirl.create(ansible_provider, :with_authentication).managers.first }
```

Comment:
I think it would be much simpler to define a let for the `provider` outside of the shared examples?